object_test.go

```diff
@@ -312,7 +312,7 @@ func (*A16) JavaClassName() string {
 }
 
 type TestObjectStruct struct {
-	Value int `hessian:"_value"`
+	Value int32 `hessian:"_value"`
```

Comment:
why changed to int32?

Reply:
```java
public Object replyObject_1() {
    return new TestObject(Integer.valueOf(0));
}
```
Java code use the `Interger` to represent `0`, so `0` will be encode as int32.

However in go `int` type is encode as int64:

https://github.com/dubbogo/hessian2/blob/adaf25425872fad0d85c75a7a81e492f5f39f7e5/encode.go#L86-L94